lib/tools/patching.py

```
@@ -303,32 +309,121 @@
-
-			if not split_patches and (committed is not None):
-				commit_hash = committed['commit_hash']
-				one_patch.git_commit_hash = commit_hash
-
-				if rewrite_patches_in_place and not (one_patch.parent.patch_dir.is_autogen_dir):
-					rewritten_patch = patching_utils.export_commit_as_patch(git_repo, commit_hash)
-					one_patch.rewritten_patch = rewritten_patch
+
+	# PARALLEL PROCESSING MODE (using overlayfs)
+	if parallel_patches:
+		log.info(f"Using parallel patch processing mode with overlayfs...")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)